apps/web-app/src/app/features/assessment/state/use-assessment-survey-questions.ts

```diff
@@ -35,11 +35,14 @@ function isQuestionToShow(assessment: UserAssessment, question: QuestionWithCate
     const hasQuestionText = () => question.question[questionForm] !== undefined;
     const valid = () => !question.question.validWhen || question.question.validWhen(context);
     const eligibleForAssessor = context.assessor.roles?.length > 0 ? () => !question.question.eligibleForAssessor || question.question.eligibleForAssessor(context) : () => true;
+    const skipped = !!assessment.verifiedCategories && assessment.verifiedCategories[question.question.id]?.status !== undefined
```

Comment:
funkcje tutaj były żeby przesunąć w runtime liczenie czegoś co wymaga doatkowych operacji ale nie jest konieczne za każdym razem. tutaj już praktycznie masz wszystko po lewej stronie ? to już bez znaczenia czy jest wydzielone :)
czy nie byłoby czytelniej np.:
```
const markedToAsk = (assessment.verifiedCategories?.[question.question.id]?.status ?? UserAssessmentVerification.Ask) === UserAssessmentVerification.Ask;
```
?

Reply:
i może nawet dać je wyżej żeby uniknąć wykonania validWhen i eligibleForAssessor jeżeli to proste przyrównanie już  może odciąć decyzję